Sources/MapboxNavigation/UIImage.swift

```diff
@@ -33,4 +33,14 @@ extension UIImage {
             (text as NSString).draw(in: rect.offsetBy(dx: 0, dy: (constrainedSize.height - font.lineHeight) / 2).integral, withAttributes: textFontAttributes)
         }
     }
+
+    func scaled(_ size: CGSize) -> UIImage? {
```

Comment:
Nit: `scaled(to:)` reads more naturally.

Reply:
Done.